chimp.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  path: "tests/spec",
+  browser: "chrome",
+  log: "debug",
+  debug: true,
+  sync: false,
+
```

Comment:
add an option to enable chai so you dont need to enable it in the run request.
`chai: true,`